crates/bevy_sprite/src/picking_backend.rs

```diff
@@ -11,25 +11,56 @@ use bevy_ecs::prelude::*;
 use bevy_image::Image;
 use bevy_math::{prelude::*, FloatExt, FloatOrd};
 use bevy_picking::backend::prelude::*;
+use bevy_reflect::prelude::*;
 use bevy_render::prelude::*;
 use bevy_transform::prelude::*;
 use bevy_window::PrimaryWindow;
 
+/// Should the backend count transparent pixels as part of the sprite for picking purposes
+#[derive(Debug, Clone, Copy, Reflect)]
+pub enum SpriteBackendAlphaPassthrough {
+    /// Even if a sprite is picked on a transparent pixel, it should still count within the backend.
+    /// Only consider the rect of a given sprite.
+    NoPassthrough,
```

Comment:
`NoPassthrough` is a bit repetitive. I'd suggest just calling this varient `Disabled` or something. The type name already includes "Passthrough" so this is currently repeating itself. e.g.
`SpriteBackendAlphaPassthrough::Disabled`

Reply:
If we really want to bikeshed this :), the type name on its own isn't super clear that it has anything to do with picking.

I think something like this is more succinct and descriptive of the two modes available to users:
```
enum SpritePickingMode {
    BoundingBox,
    AlphaThreshold(u8)
}
```

Reply:
Hey, bikeshed away. I'll sort it out tomorrow 😅

Reply:
Renamed 😄